docs/validator/delegation.md

```diff
@@ -14,7 +14,7 @@ Disclaimer: the list below is community-maintained, and is not an endorsement by
 | Provider  | URL (to copy and paste)  | Added on |
 | --------- | ------------------------ | -------- |
 | NEAR Wallet | https://wallet.near.org | work in progress |
-| add here | your link | first come, first serve |
+| DokiaCapital | https://staking.dokia.cloud | first come, first serve |
```

Comment:
@awrelll Could you resubmit, but leave in the row `| add here | your link | first come, first serve |`? That way people will still be able to see that as an instructional indicator. Also, could you remove the `first come, first serve` in the `Added on` column  of your row and put today's date?

Thanks so much for adding your organization!